Resources/Prototypes/_LostParadise/Entities/Clothing/Head/helmet.yml

```
@@ -137,3 +137,169 @@
+  - type: ExplosionResistance
+    damageCoefficient: 0.9
+  - type: PressureProtection
+    lowPressureMultiplier: 100
+  - type: BreathMask
+  - type: Item
+    heldPrefix: open
+
+- type: entity
+  parent: ClothingHeadBase
+  id: LPPClothingHeadHelmetSwat
+  categories: [ HideSpawnMenu ]
```

yamllint (1.35.1)

[error] too many spaces inside brackets

(brackets)